app/assets/stylesheets/frontend/views/_how-gov-works.scss

```
@@ -223,6 +235,18 @@
         }
       }
     }
+    .prime-minister {
+      .one-half {
+        width: $one-third;
+        &:nth-child(2) {
+          width: $one-third;
+        }
```

I think that the `nth-child(2)` selector isn't needed, as it inherits from `.one-half`. Toggling it off in dev tools in FF, Chrome, and Safari doesn't affect the layout.
```suggestion
        width: $one-third;
```